Makefile

```diff
@@ -1,33 +1,48 @@
+NAMESPACE := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["namespace"])')
+NAME := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["name"])')
+VERSION := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["version"])')
+MANIFEST := build/collections/ansible_collections/$(NAMESPACE)/$(NAME)/MANIFEST.json
+
+PLUGIN_TYPES := filter modules module_utils doc_fragments
```

Comment:
Should i add 'inventory' and 'callback' now?

Reply:
Can't you just dynamically gather that with `$(wildcard plugins/*)` or similar?

I think the following should do it:
```make
PLUGIN_TYPES := $(filter-out __%,$(notdir $(wildcard plugins/*)))
```

/me goes to wash his hands

Reply:
(yes, this is, again dependant on the fact that one does not pass the trailing slash, shoot me)

Reply:
That is if we rely on the fact that each directory here is in fact a plugin type.
But yes, why not. I will try this.
We can also use the filter-out trick in the other wildcard thingies.

Reply:
I would consider every directory under `plugins` to be a plugin type, yes.